orion/sites/admin/src/programs/program.js

```diff
@@ -22,24 +22,52 @@ export class ProgramPage extends React.Component {
       });
   }
 
-	render() {
+  onClickSelectAll() {
+      var items = document.getElementsByName("unpublished");
+      for (var i=0; i<items.length; i++) {
+          if (items[i].type == 'checkbox') {
+              items[i].checked = true;
+          }
+      }
+  }
+
+  render() {
     const rows = this.state.list.map((row, index) => {
       return <ProgramRow key={index} row={row}/>
     });
     const numRows = rows.length;
-		return (
+    let numUnpublished = 0;
+    let numSelected = 0;
+	return (
       <div id="view-program">
-        <h1>All Programs ({numRows})</h1>
+        <ul>
+            <h1>All Programs ({numRows}) </h1>
+            <div> You have {numUnpublished} unpublished items. </div>
+            <div> You have selected {numSelected} items to publish. </div>
+        </ul>
         <ul id="list-heading">
+          <button className="li-small" onClick={this.onClickSelectAll}>Select All</button>
           <li className="li-med">ProgramKey</li>
           <li className="li-med">Name</li>
           <li className="li-small">Grade1</li>
           <li className="li-small">Grade2</li>
         </ul>
         <ul id="list-rows">
+            <script>hideInputs()</script>
           {rows}
         </ul>
-        <Link className="add-program" to={"/programs/add"}>Add Program</Link>
+        <ul id="list-buttons">
+            <div className="li-med">
+                <button>
+                <Link className="add-program" to={"/programs/add"}>Add Program</Link>
+                </button>
+            </div>
+            <div className="li-med">
+                <button>
+                <Link className="publish" to={"/programs/add"}>Publish</Link>
```

Comment:
The Publish button won't be a link. It will be an API call like in the edit page, so the button will have an onClick.

For now don't worry about the API call itself, but just make this button use an onClick